### DISCUSSION TITLE
GitHub Education Re-verification Problem

### DISCUSSION THREAD

**Discussion by iamprincerajv** [community/community#140603] [GitHub Education]
### Hi everyone,

I received an email from Github Education to re-verify myself as student but whenever i click on the link given (https://education.github.com/discount_requests/application). I get redirected to https://github.com/pricing and there is no option to renew or re verify myself. Moreover, they said that i have 30 days to re-verify and until then my services won't be interrupted but my copilot stopped working right away. What to do ?

**Comment by casualdevvv** [comment]
I'm going through the same, I'm pretty sure AFTER it expires you can then reapply

**Reply by casualdevvv** [reply]
My bad maybe I misread that one 😭

**Comment by Ahsankhalid618** [comment]
@iamprincerajv Just click on [Github-Education-Students](https://github.com/education/students) and verify your email address, and billing information (No need to add a credit card, Just add your Full Name as on your ID card) and verify your benefits. Make sure you are eligible for the benefits as per the GitHub education program. I hope it helps.
